easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED.eb

```diff
@@ -0,0 +1,41 @@
+name = 'CP2K'
+version = '20150904'
+versionsuffix = '-PLUMED'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+# we use the git mirror: this is r15816 or 68331ef35b7
+sources = ['68331ef35b7.zip']
+source_urls = ['https://github.com/cp2k/cp2k/archive/']
+
+patches = [
+    'CP2K-2.6.0-ifort-compiler-bug-fix.patch',
+    'CP2K-2.4.0-fix_compile_date_lastsvn.patch',
+]
+
+dependencies = [
+    ('Libint', '1.1.4'),
+    ('libxc', '2.2.1'),
+    ('PLUMED', '2.1.3'),
+]
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.2'),
+]
+
+# don't use parallel make, results in compilation failure
+# because Fortran module files aren't created before they are needed
+parallel = 1
```

Comment:
I don't think this is needed anymore, should work without it (I was told a while ago that parallel building works fine for recent CP2K versions)

so, let's chuck it and see what happens